pkgs/servers/monitoring/grafana/default.nix

```diff
@@ -2,33 +2,24 @@
 
 buildGoModule rec {
   pname = "grafana";
-  version = "8.0.2";
+  version = "8.0.3";
 
   excludedPackages = [ "release_publisher" ];
 
   src = fetchFromGitHub {
     rev = "v${version}";
     owner = "grafana";
     repo = "grafana";
-    sha256 = "sha256-kCsrLZ0EbuMwqqDvUvhm8+B/vh6FpeJ5zkwste+qwyQ=";
+    sha256 = "sha256-GGtmsz3c7Q6mEGCJ6cdz2CjOW0ovZZW8j6LMfFgrMZ4=";
   };
 
   srcStatic = fetchurl {
     url = "https://dl.grafana.com/oss/release/grafana-${version}.linux-amd64.tar.gz";
-    sha256 = "sha256-QBMGLN3MjYJcv2vbs9GHfrCixcV7nH+Ox3o6/YtRYak=";
+    sha256 = "sha256-2x0FhKinrXAFenJcUDh4Q3RJNBrqixKBNZT7BZNNOj8=";
   };
 
   vendorSha256 = "sha256-x7sSVIim/TOhMTbnRK/fpgxiSRSO8KwGILTE2i1gU3U=";
 
-  patches = [
-    # https://github.com/grafana/grafana/pull/35635 (fixes declarative plugins for us)
-    (fetchpatch {
```

Comment:
Please remove the unused fetchpatch from inputs.

Reply:
@SuperSandro2000 done